sqld/src/rpc/wal_log.rs

```diff
@@ -88,6 +89,12 @@ impl WalLog for WalLogService {
             let mut guard = self.replicas_with_hello.write().unwrap();
             guard.insert(replica_addr);
         }
-        Ok(tonic::Response::new(self.logger.hello_response().clone()))
+        let response = HelloResponse {
+            database_id: self.logger.database_id.to_string(),
```

Comment:
Unless someone else is using database_id, maybe we could just store it as string? I know it's not performance critical but there's no cost in storing it as string so it's better to have more performant code.